### DISCUSSION TITLE
Unable to install - SSH disconnects after entering email address for django and meshcentral

### DISCUSSION THREAD

**Discussion by onlinegill** [amidaware/tacticalrmm#1804] [General]
Hello, 

I am trying to install on contabo VPS with 4GB ram and i have tried multiple OS's and when it  ask enter email address for django and meshcentral than SSH connection gets disconnected.  tried with and without firewall and allowed all SSH connections.

**Comment by dinger1986** [comment]
What happens if you sue their console? This isn't an issue that's been reported before so not an issue with the installer script.

You could also try https://mobaxterm.mobatek.net/download.html
